test/unit/sagas/clients.js

```diff
@@ -44,13 +44,14 @@ test('createSnapshot()', (t) => {
   });
 });
 
-test('exportGist()', (t) => {
+test('exportProject()', (t) => {
   const url = 'https://gist.github.com/abc123';
+  const exportType = 'gist';
```

Comment:
So for the most part, since generally all export types are treated the same, it’s totally fine/good to only test one export type. However in the case of the saga, the saga’s behavior is in fact quite different depending on the export type, so we should write tests for all three scenarios here.

Reply:
Yeah this makes sense. Will update the tests!